packages/inter-protocol/test/smartWallet/boot-psm.js

```diff
@@ -262,13 +260,13 @@ export const buildRootObject = (vatPowers, vatParameters) => {
       ),
       makeVatsFromBundles(powersFor('makeVatsFromBundles')),
       buildZoe(powersFor('buildZoe')),
-      makeBoard(powersFor('makeBoard')),
+      makeBoard(allPowers),
```

Comment:
> why are they given allPowers?
Because I took them out of the manifest, so there's no permit.

> this shouldn't be in production.

Well, this is test code now, right?

Reply:
Indeed